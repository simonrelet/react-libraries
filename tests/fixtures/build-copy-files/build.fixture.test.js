'use strict'

const fixture = require('../../setupTests')

describe(`${fixture.fixtureName} build`, () => {
  it('should not fail', async () => {
    const result = await fixture.runScript('build', [
      '-c',
      'src/**/*.copy.json',
    ])

    expect(result).toMatchSnapshot()
  })

  it('should ignore files during build', async () => {
    expect(await fixture.listFolderContent('build')).toEqual([
      'cjs/aaa.js',
      'cjs/bbb.copy.json',
      'cjs/bbb.js',
      'cjs/ccc/ccc.copy.json',
      'cjs/ccc/ccc.js',
      'es/aaa.js',
      'es/bbb.copy.json',
      'es/bbb.js',
      'es/ccc/ccc.copy.json',
      'es/ccc/ccc.js',
    ])
  })
})
