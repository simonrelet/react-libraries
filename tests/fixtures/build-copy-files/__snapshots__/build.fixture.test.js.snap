// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`build-copy-files build should not fail 1`] = `
Exit code: 0

--- Output ---
$ react-libraries build -c 'src/**/*.copy.json'
CommonJS build successful.
ES modules build successful.
--------------
`;
